app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -125,16 +144,26 @@ export class SemiCircle extends React.Component {
     100% { offset-distance : ${arcEndPercent}% }
     `;
 
+    const cityImage = getCityImage(this.props.cityName);
+
     const styleRadialGradient = () => {
       if (nightMode) return colors.lightBlue;
+      if (cityImage === bldImage) return colors.lightYellow;
       return colors.lightPink;
     };
 
     const styleLinearGradient = () => {
       if (nightMode) return colors.blue;
+      if (cityImage === bldImage) return colors.yellow2;
```

Comment:
How does BLD look in night mode?

Reply:
Heres a screenshot of what it looks like. I think it looks similar to PVD in night mode, but can make adjustments if needed! 

![Screen Shot 2021-07-06 at 12 04 28 PM](https://user-images.githubusercontent.com/49659834/124646828-6221d100-de52-11eb-98a9-0477d9ecf415.png)

Reply:
:+1: